openmc/mgxs/mgxs.py

```diff
@@ -4847,15 +4405,15 @@ class MultiplicityMatrixXS(MatrixMGXS):
 
     """
 
-    def __init__(self, domain=None, domain_type=None,
-                 groups=None, by_nuclide=False, name='', num_polar=1,
-                 num_azimuthal=1):
+    def __init__(self, domain=None, domain_type=None, groups=None,
+                 by_nuclide=False, name='', num_polar=1, num_azimuthal=1):
         super(MultiplicityMatrixXS, self).__init__(domain, domain_type, groups,
                                                    by_nuclide, name, num_polar,
                                                    num_azimuthal)
         self._rxn_type = 'multiplicity matrix'
         self._estimator = 'analog'
         self._valid_estimators = ['analog']
+        self.nu = True
```

Comment:
If we decide to remove `nu` as a property of the abstract `MGXS` base class, I think it can be removed here too.